examples/isentropic_example.py

```diff
@@ -157,7 +146,10 @@
 
 
 # Calculate Montgomery Streamfunction and scale by 10^-2 for plotting
-msf = mpcalc.montgomery_streamfunction(isenthgt, isenttmp) / 100.
+msf = mpcalc.montgomery_streamfunction(
+    isent_data['Geopotential_height'],
+    isent_data['temperature']
+).values / 100.
```

Comment:
Nope, I only did it that way since it was how it was done originally. I'd vote to punt on this to the 1.0 documentation work, since this example could probably use some further cleanups to better leverage xarray integration (I only did the bare minimum to get the docs passing while using `isentropic_interpolation_as_dataset`)